mage_ai/data_cleaner/statistics/calculator.py

```diff
@@ -176,16 +168,19 @@ def __statistics_overview(self, series, col):
             if series.size == 0
             else series.isnull().sum() / series.size,
             f'{col}/null_value_count': series.isnull().sum(),
-            f'{col}/max_null_seq': self.get_longest_null_seq(series),
             f'{col}/value_counts': df_top_value_counts.to_dict(),
         }
 
+        data[f'{col}/max_null_seq'] = (
```

Comment:
do we need to calculate this for columns with all null values?

Reply:
Don't have to calculate, should've been `count()` instead of `size` there. Fixed now.